scripts/fonts.json

```diff
@@ -61,6 +62,10 @@
       [6928, 6975],
       [71216, 71231]
     ],
+    "cherokee": [
+      [5024, 5119],
+      [43888, 43967]
```

Comment:
Thanks for including this range. I’m reading conflicting things about whether a case distinction is meaningful in real-world Cherokee usage, but it shouldn’t be a problem to have this extra glyph PBF available in case lowercase characters ever appear in OSM.